src/server/pfs/server/server.go

```diff
@@ -18,26 +16,14 @@ const (
 	GoogleBackendEnvVar = "GOOGLE"
 )
 
-// APIServer represents an api server.
+// APIServer represents and internal api server.
 type APIServer interface {
-	pfsclient.APIServer // SJ: WOW this is bad naming
-	shard.Frontend
+	pfsclient.APIServer // SJ: also bad naming
 }
```

Comment:
Do we even need this type anymore? Can't we just use `pfsclient.APIServer`? Also the comment is wrong now I think.